### ISSUE TITLE
Pseudoautosomal inheritance

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7683]
**Preferred term label:**
Pseudoautosomal inheritance

**Synonyms**
Pseudoautosomal region inheritance

**Definition (free text, please give PubMed ID)**
A pattern of inheritance observed for alleles in the X-Y identical regions is referred to as pseudoautosomal inheritance, because it resembles the pattern seen for alleles located on autosomes. 
Comment: The pseudoautosomal regions (PAR1 and PAR2) are short regions of homology between the mammalian X and Y chromosomes. The PAR behave like an autosome and recombine during meiosis. Thus genes in this region are inherited in an autosomal rather than a strictly sex-linked fashion.
PMID:18660847

@angharadroberts
I am adding this new term to be a parent of the two new proposed terms for the PAR inheritance (mono and bi allelic)

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term as HP:0034339
please reopen as required.
